test/python/transpiler/test_inverse_cancellation.py

```diff
@@ -0,0 +1,124 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Testing inverse_cancellation
+"""
+from numpy import pi
```

Comment:
As a convention, we usually import numpy under the `np` alias. Then, rather than referencing `pi`, reference `np.pi`. This is nice because it tells us what external library we are using for this value.
```suggestion
import numpy as np
```